cypress/support/config/settings.js

```diff
@@ -7494,15 +7494,15 @@ module.exports = () => ({
       storyPage: {
         environments: {
           live: {
-            paths: [],
+            paths: ['/ukrainian/features-53779581'],
             enabled: false,
           },
           test: {
-            paths: [],
+            paths: ['/ukrainian/23154182'],
             enabled: false,
           },
           local: {
-            paths: [],
+            paths: ['/ukrainian/23340963'],
```

Comment:
Are you also planning to set `enabled: true` here?

Reply:
@karinathomas doing that ensures cypress tests run against that asset right ?

Reply:
not sure if this is STY specific, but other page types seem to have this set to `true` except some STY's

Reply:
Yeah - probably a discussion to have with QA - would they like to have these URLs tests in E2Es on AWS?

Reply:
most of those we have for STY's aren't enabled yet, is this something we'd like to turn on ? cc @joshcoventry @jimjohnsonrollings @andrewscfc

Reply:
Didn't we decide not to run E2Es for all page types as that took too long? I think we chose to do one LTR, one RTL and a few more for other services with significant differences, but I can't remember all of them.